package.json

```diff
@@ -52,6 +52,8 @@
     ]
   },
   "dependencies": {
-    "ws": "^7.1.2"
+    "cross-fetch": "^3.0.4",
+    "querystring": "^0.2.0",
```

Comment:
this package is no longer maintained (since 2 years)

ok, with the 2 others.

Reply:
Ah, good catch. Originally went with that package because it had the same API as node's built-in querystring. I'll modify the REST transport to use [`query-string`](https://www.npmjs.com/package/query-string) instead, which looks pretty active.